components/footer/index.css

```
@@ -1,5 +1,20 @@
 .footer {
-    background-color: var(--chocolate);
-    height: 80px;
+    height: 144px;
     min-height: 80px;
+    color: var( --gray );
+    margin: auto;
 }
+
+.footer_container {
```

```suggestion
.footer__container {
```

Rename the class according to BEM